app/services/feature_service.rb

```diff
@@ -0,0 +1,11 @@
+module FeatureService
+  class << self
+    def enabled?(feature_name)
+      return false if Settings.features.blank?
+
+      # Supports nested feature segments
```

Comment:
Can we add a test for this?

Reply:
done

Reply:
Can we have a test for the nested feature splitting?

Reply:
done https://github.com/DFE-Digital/register-trainee-teacher-data/pull/44/commits/3ee1d0f6d02cca943c6fc1c97ee8f5513ba62319